docs/stack/insight/_index.md

```diff
@@ -32,11 +32,15 @@ Browse, filter and visualize your key-value Redis data structures.
 * CRUD support for [RedisJSON](https://oss.redis.com/redisjson/)
 * Group keys according to their namespaces
 
-<img src="/docs/stack/insight/images/Browser.png">
+  <img src="/docs/stack/insight/images/Browser.png">
+
+* View, validate, and manage your key values using new formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII)
+
+  <img src="/docs/stack/insight/images/data_formatting.png">
```

Comment:
@nermiller , let's use new images with higher resolution.
Please let me know if you need them.

Reply:
@ViktarStarastsenka yes, please, point me to them. Thanks!

Reply:
View, validate, and manage your key values in a human-readable format using formatters that prettify and highlight data in different formats (for example, Unicode, JSON, MessagePack, HEX, and ASCII) in the Browser tool.